compiler_opt/es/blackbox_optimizers.py

```diff
@@ -897,12 +914,12 @@ def update_hessian_part(self, perturbations, function_values, current_value,
         self.saved_function_values = np.append(self.saved_function_values,
                                                function_values)
 
-  def create_hessv_function(self):
+  def create_hessv_function(self) -> Callable:
```

Comment:
ditto

Reply:
The function details have been added now